fairseq/utils.py

```diff
@@ -248,6 +249,29 @@ def load_align_dict(replace_unk):
     return align_dict
 
 
+def print_embed_overlap(embed_dict, vocab_dict):
+     embed_keys = set(embed_dict.keys())
+     vocab_keys = set(vocab_dict.symbols)
+     overlap = len(embed_keys & vocab_keys)
+     print("| Found {}/{} types in embedding file.".format(overlap, len(vocab_dict)))
+
+def parse_embedding(embed_path):
+    embed_dict = dict()
+    with open(embed_path) as f_embed:
+        for line in f_embed:
```

Comment:
I think it'd be more generally useful to adhere to the fastText vector format, i.e., skip the header row. Something like (not tested):
```
with open(embed_path) as f_embed:
    header = next(f_embed)  # num_embed, embed_dim
    for line in f_embed:
        (...)
```